pyracing/client.py

```diff
@@ -1,12 +1,10 @@
 from . import constants as ct
-from .responses.last_races_stats import LastRaceStats
-from .responses.career_stats import CareerStats
-from .responses.yearly_stats import YearlyStats
-from .responses.chart_data.chart_data import ChartData
-from .responses.chart_data.irating import IRating
-from .responses.chart_data.ttrating import TTRating
-from .responses.chart_data.license_class import LicenseClass
-from .responses.season import Season
+from .response_objects import career_stats
```

Comment:
These could be in one line. `from .response_objects` import career_stats, chart_data, iracing_data` and so on